ros/lib/utils.py

```diff
@@ -105,3 +106,8 @@ def sort_io_dict(performance_utilization: dict):
     performance_utilization.update({**sorted_io_dict})
     del performance_utilization['io']
     return performance_utilization
+
+
+def default_queries(account_number):
```

Comment:
```suggestion
def system_ids_by_account(account_number):
```
In my opinion, as it handles account filter i.e main scope it better to rename it to something specific.